### PR TITLE
Turkish translation was created

### PR DESCRIPTION
**Pull request by serkan-maker**
I finished the translation in Transifex. Can you check it?

### CODE REVIEW

File: locale/retext_tr.ts [hunk 1]
```diff
@@ -0,0 +1,849 @@
...
+        <location filename="../ReText/tab.py" line="482"/>
+        <source>Create missing file?</source>
+        <translation>Eksik dosya oluşturulsun mu?</translation>
+    </message>
+    <message>
+        <location filename="../ReText/tab.py" line="482"/>
+        <source>The file &apos;%s&apos; does not exist.
+
+Do you want to create it?</source>
+        <translation>&apos;% S&apos; dosyası mevcut değil.
```
Comment (mitya57) [inline locale/retext_tr.ts:295]:
```suggestion
        <translation>&apos;%s&apos; dosyası mevcut değil.
```

File: locale/retext_tr.ts [hunk 2]
```diff
@@ -0,0 +1,849 @@
...
+    </message>
+    <message>
+        <location filename="../ReText/window.py" line="1289"/>
+        <source>Select one or several images to open</source>
+        <translation>Açmak için bir veya birkaç resim seçin</translation>
+    </message>
+    <message>
+        <location filename="../ReText/window.py" line="1307"/>
+        <source>ReText %s (using PyMarkups %s)</source>
+        <translation>ReText %s (using PyMarkups %s)</translation>
```
Comment (mitya57) [inline locale/retext_tr.ts:794]: Here you can translate the word ‘using’.

File: locale/retext_tr.ts [hunk 3]
```diff
@@ -0,0 +1,849 @@
...
+        <source>Set as default</source>
+        <translation>Öntanımlı olarak ata</translation>
+    </message>
+</context>
+<context>
+    <name>Proxy</name>
+    <message>
+        <location filename="../ReText/fakevimeditor.py" line="61"/>
+        <source>No write since last change (add ! to override)</source>
+        <translation>Son değişiklikten bu yana yazma yok (geçersiz kılmak için! Ekleyin)</translation>
```
Comment (mitya57) [inline locale/retext_tr.ts:247]: Here the string means that one should literally type an exclamation mark. Did you translate it correctly? It is suspicious to me that `Ekleyin` is capitalized. And maybe add a space before `!`?

**Review by mitya57 — CHANGES_REQUESTED** [review]
Thank you very much for your translation!

I usually grab translations from Transifex before every release (or once in a few months), but as you submitted a PR, I will review it here. Left some comments.

**Comment by serkan-maker** [comment]
Thank you for this program. yes you are right the translation is wrong. i applied the fix, now i install it.
[retext_tr.ts.zip](https://github.com/retext-project/retext/files/6331974/retext_tr.ts.zip)

**Review by mitya57 — APPROVED** [review]
Thank you! Now it looks good, I will merge it a bit later.

**Comment by mitya57** [comment]
Merged, thanks again. Now I am looking at the translation in desktop file.

It currently says:

> Markdown ve Yeniden Yapılandırılmış Metin için basit ama güçlü düzenleyici

Do you think reStructuredText should be translated? It is a proper name, not two words. See <https://en.wikipedia.org/wiki/ReStructuredText>.

So maybe better to make it “Markdown ve reStructuredText için basit ama güçlü düzenleyici”?

Or just “Markdown ve reStructuredText için basit düzenleyici”, as “but powerful” was not part of the original string here.

**Comment by serkan-maker** [comment]
I corrected in transifex

**Comment by mitya57** [comment]
Applied in b39ad26a32a4930b. Thanks.

**Pull request merged**